### ISSUE TITLE
pytorch 1.7.1 throws a warning on nn.functional.interpolate()

### ISSUE THREAD

**Issue by xvdp** [facebookresearch/dino#119]
nothing major, just a warning on interpolate
Since this code base was done on 1.7.1 I presume the default behaviour is
`nn.functional.interpolate(...,recompute_scale_factor=False)`
so code should fixed accordingly

`[...]/python3.9/site-packages/torch/nn/functional.py:3103: UserWarning: The default behavior for interpolate/upsample with float scale_factor changed in 1.6.0 to align with other frameworks/libraries, and now uses scale_factor directly, instead of relying on the computed output size. If you wish to restore the old behavior, please set recompute_scale_factor=True. See the documentation of nn.Upsample for details. 
  warnings.warn("The default behavior for interpolate/upsample with float scale_factor changed "`

**Comment by mathildecaron31** [comment]
Yes I also get these warnings when training, nothing to worry about.
